Interfaces/API/test/Test_JobAPI.py

```diff
@@ -0,0 +1,60 @@
+""" Basic unit tests for the Job API
+"""
```

Comment:
There's already a "DIRAC.Interfaces.API.test.Test_API" file that has a basic test of the Job API. Can you please merge the 2? Thanks.